ReactSkia/BUILD.gn

```diff
@@ -213,6 +216,16 @@ source_set("ReactSkia") {
     "LegacyNativeModules/uimanager/UiManagerModule.h",
   ]
 
+  # Activity Indicator Native Component
+  sources += [
```

Comment:
should this be similar to how textinput files were added as rn_cxx_component

Reply:
addressed